VitvitskyAkim/HW5/Task5_3.py

```diff
@@ -0,0 +1,7 @@
+n=int(input("The factorial of what number do you want to know?: "))
+k=1
+j=1
```

Comment:
look at this implementation:
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)